### ISSUE TITLE
Legacy Mode needed for input fields while Options.plist doesn't seem to be supported in Legacy Mode

### ISSUE THREAD

**Issue by N-I-N-0** [Matchstic/Xen-HTML#115]
**Describe the bug**
1. Options.plist doesn't work in Legacy Mode
2. it's not possible to write into html inputs without Legacy Mode activated

**To Reproduce**
1. Try accessing information from Options.plist while Legacy Mode is activated
2. Try typing in input field while Legacy Mode isn't activated

**Expected behavior**
Typing should be possible without Legacy Mode while the Options.plist should be accessible in Legacy Mode

**Smartphone (please complete the following information):**
 - Device: iPhone X
 - OS: iOS 12.1.2

**Additional context**
Thanks for your time :)

**Comment by Matchstic** [comment]
Can you give a link to a widget where input doesn't work please? That will help me tremendously to fix this

**Comment by N-I-N-0** [comment]
I'm currently working on one myself. Will upload it in a minute ^^
Thx for your time

**Comment by N-I-N-0** [comment]
[Archiv.zip](https://github.com/Matchstic/Xen-HTML/files/3269366/Archiv.zip)

**Comment by N-I-N-0** [comment]
[https://www.dropbox.com/s/vd7gy2yz41tn06o/com.NIN0.XenSearch.deb?dl=1](url)
Now it's a .deb ...
I would like to publish it, but I don't have an own repo -> so I would like to ask: would you host this widget until I have my own repo? You don't have to but it would be nice ^^

**Comment by Matchstic** [comment]
Thanks for the link, I’ll look at it when I can.

As to hosting, my repo really isn’t setup to have other people’s debs on there; you’d be forced to wait for me (sometimes days/weeks) just to get around to pushing updates etc. Sorry!

**Comment by N-I-N-0** [comment]
Well thanks anyway.
Just created a repo with GitHub now: https://n-i-n-0.github.io/repo/
I'll update the index.html and the description of XenSearch in the next few days/weeks. Is there a way to get my repo linked in XenHTML?

**Comment by Matchstic** [comment]
@N-I-N-0 To get your repo linked, I need the following details:

- URL of your repo
- An icon for your repo, at least 120x120 pixels

Then, I update a plist on my server, which then gets downloaded in the Settings panel.

**Comment by N-I-N-0** [comment]
Thanks for the reply. Here are the URL and the icon:
URL: https://n-i-n-0.github.io/repo/
Icon: ![5E229C9F-0F64-4A5E-90BD-6F2C8EB6C22E](https://user-images.githubusercontent.com/26253311/61182010-f2dfc080-a62d-11e9-8eba-b67152ef2dd7.JPEG)

✌️

**Comment by Matchstic** [comment]
@N-I-N-0 Done, check the Settings page in Xen HTML!

**Comment by N-I-N-0** [comment]
Thanks 😃✌️

**Comment by Matchstic** [comment]
Currently looking into this issue. I've so far found a bug that prevented touches on widgets in some cases, which I've fixed. 

As for the keyboard, I'm able to observe that tapping the widget does bring up the keyboard, which is then instantly dismissed... not cool! 

Options.plist not working in Legacy Mode is something I'm ignoring, because once I've implemented #124 , I'll be removing Legacy Mode because it won't be needed any longer.
